client/src/components/BasicMath/Login.js

```diff
@@ -35,7 +35,7 @@ class Login extends Component {
     const { ss: { i18n } } = window;
 
     if (!numbers) {
-      return <LoadingIndicator />;
+      return <div className="mfa-loader"><LoadingIndicator /></div>;
```

Comment:
I should have made this class a little more distinct from the one in `LoadingIndicator`, which is `mfa__loader`. This class _is_ necessary, as it replaces a bit of the styling that used to be baked into `LoadingIndicator` but which I needed to remove to make it usable more broadly.

It might be best to change the class on `LoadingIndicator` instead - to `.loading-indicator`, since it's an independent component. Thoughts?

Reply:
You could also make the class name extensible via a passed prop

Reply:
Ok I think we're onto something in the discussion below: https://github.com/silverstripe/silverstripe-mfa/pull/127#discussion_r286701883